escrow/free_call_payment_handler.go

```diff
@@ -71,17 +71,29 @@ func (h *freeCallPaymentHandler) getPaymentFromContext(context *handler.GrpcStre
 		return
 	}
 
+	authTokenIssueDate, err := handler.GetBigInt(context.MD, handler.FreeCallAuthTokenBlockNumberHeader)
```

Comment:
Better to rename `authTokenIssueDate` to `authTokenExpiryDate`
And `FreeCallAuthTokenBlockNumberHeader` to `FreeCallAuthTokenExpiryBlockNumberHeader`